tasty-query/shared/src/main/scala/tastyquery/Flags.scala

```diff
@@ -78,16 +78,10 @@ private[tastyquery] object Flags:
   val SignaturePolymorphic: Flag = newFlag("SignaturePolymorphic")
   val SuperParamAlias: Flag = newFlag("SuperParamAlias")
   val Static: Flag = newFlag("Static")
-  private[tastyquery] val StableRealizable: Flag = newFlag("StableRealizable")
+  val StableRealizable: Flag = newFlag("StableRealizable")
```

Comment:
im sure this was private because it was flaky if it actually appears in tasty

Reply:
The whole `Flags` is private now. 😉